docs/articles_en/openvino-workflow/torch-compile.rst

```diff
@@ -79,7 +107,9 @@ You can also set OpenVINO specific configuration options by adding them as a dic
 Windows support
 +++++++++++++++++++++
 
-Currently, PyTorch does not support ``torch.compile`` feature on Windows officially. However, it can be accessed by running
+PyTorch supports ``torch.compile`` officially on Windows from version 2.3.0 onwards.
+
+For PyTorch versions below 2.3.0, ``torch.compile`` feature is not supported on Windows officially. However, it can be accessed by running
 the below instructions:
```

Comment:
```suggestion
For PyTorch versions below 2.3.0, the ``torch.compile`` feature is not supported on Windows
officially. However, it can be accessed by running the following instructions:
```